.github/pull_request_template.md

```diff
@@ -15,10 +15,35 @@
 <!-- Describe what manual testing you did (if no testing was done, explain why). -->
 
 ### Automated Testing
-<!--If you added/updated unit/integration/validation tests, describe what cases they cover and do not cover. -->
+<!-- Ensure there are unit/integration/validation tests added (if possible); describe what cases they cover and do not cover. -->
+* Test types added/modified:
+    * Unit
+    * Integration (Go Framework)
+    * Integration (v2prov Framework)
+    * Validation (Go Framework)
+    * Other - Explain: _EXPLAIN_
+    * None
+    * _REMOVE NOT APPLICABLE OPTIONS AND THIS LINE_
+* If "None" - Reason: _EXPLAIN THE REASON_
+  <!-- 
+  Non-exhaustive list of reasons:
+    - Lack of the framework capable of testing this fix/change
+    - Tight deadlines / critical priority to get fix/change in - !ensure GH issue is logged to add tests!
+    - No application logic is modified by this change, e.g. refactoring/cosmetic/non-code/test change
+    - Tests implemented in another PR elsewhere - !ensure GH PR link is added!
+    - Other (explain)
+  Note: Outside of the exceptions above, the "existing tests cover the changes" is very unlikely to be an acceptable reason as the existing tests generally don't cover the logic changes implemented by this PR 
+  -->
+* If "None" - GH Issue/PR: _LINK TO GH ISSUE TO ADD TESTS_
+
+Summary: _FILL OUT_
```

Comment:
Can we change "Fill out" to "Todo" or just leave blank. The fill-out term seems like its leftover from something else.

Reply:
Sure, I can change that - agreed that at the very least it's shorter and `TODO` is more familiar to engineers as that's how todo things are marked in code.